src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,96 @@
+package duke.storage;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath = "";
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void loadFile(ArrayList<Task> task) throws FileNotFoundException {
```

Comment:
you should try and improve SLAP here. many things seem repetitive, so can you extract it and parametrize appropriately?